lib/middleware/anonymous_cache.rb

```diff
@@ -73,7 +91,7 @@ def is_crawler?
       end
 
       def cache_key
-        @cache_key ||= "ANON_CACHE_#{@env["HTTP_ACCEPT"]}_#{@env["HTTP_HOST"]}#{@env["REQUEST_URI"]}|m=#{is_mobile?}|c=#{is_crawler?}|b=#{has_brotli?}|t=#{theme_ids.join(",")}#{GlobalSetting.compress_anon_cache}"
+        @cache_key ||= "ANON_CACHE_#{@env["HTTP_ACCEPT"]}_#{@env["HTTP_HOST"]}#{@env["REQUEST_URI"]}|m=#{is_mobile?}|ie=#{is_ie?}|c=#{is_crawler?}|b=#{has_brotli?}|t=#{theme_ids.join(",")}#{GlobalSetting.compress_anon_cache}"
```

Comment:
Looks good, but I wonder if we should try and keep all ie-related logic in the plugin 🤔? On the other hand, putting it here is much more obvious, and less likely to cause confusion in future. 

If we keep it like this, I think we should add a comment on the `is_ie?` method explaining why it's there.

Reply:
I've decided to make a PR for this: 

https://github.com/discourse/discourse/pull/8455